### PR TITLE
Feature/rub 515

### PR DESCRIPTION
**Pull request by ishwari-gaikwad2**
Test class for RUb-449

### CODE REVIEW

File: force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+    Test.stopTest();
+    Opportunity opp = [
+      SELECT Id, Pre_Approval_list__c
+      FROM Opportunity
+      WHERE Id = :opps2.get(0).Id
+    ];
+    system.assertEquals('Pass', opp.Pre_Approval_list__c);
+  }
+  @isTest
+  public static void handleCreditCardFinalAssessmentEvent() {
```
Comment (RemarioCRM) [inline force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls:93]: each test method. must have comments

File: force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+    Test.startTest();
+    ONeJNEventSubmissionHandler.HandleEventSubmission(Flilist);
+    OneJNOutputEventHandler.HandleEvent(Flilist1);
+    Test.stopTest();
+    Opportunity opp = [
+      SELECT Id, Final_Assessment_Decision_list__c
+      FROM Opportunity
+      WHERE Id = :opps2.get(0).Id
+    ];
+    system.assertEquals('Referred', opp.Final_Assessment_Decision_list__c);
```
Comment (RemarioCRM) [inline force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls:178]: use constants forall texts

Comment (RemarioCRM) [reply force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls:178]: picklist values

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls (hunk 2 above)
Comment (truptizende-Thinqloud) [reply force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls]: Used constants to all texts.

File: force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls [hunk 3]
```diff
@@ -0,0 +1,413 @@
...
+@isTest
+public class TestONeJNEventSubmissionHandler {
+  /**
+   * @description test method for credit card pre-assessment submission
+   * @author Trupti Zende (Thinqloud) | 06-29-2021
+   **/
+  @isTest
+  public static void handleCreditCardPreAssessmentEvent() {
+    Map<String, Id> mappedOppRecTypes = Util.getRecordTypeids(
+      OPPORTUNITYSTRING
```
Comment (RemarioCRM) [inline force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls:20]: This is failing, please rerun, report progress. then push again

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls (hunk 3 above)
Comment (ishwari-gaikwad2) [reply force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls:20]: the test class is not failing in dev box 4.

Comment (RemarioCRM) [reply force-app/main/default/classes/TestONeJNEventSubmissionHandler.cls:20]: OPPORTUNITYSTRING should be Constants. OPPORTUNITYSTRING, so yes its an error.

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

**Review by RemarioCRM — APPROVED** [review]

**Pull request merged**